sapi/phpdbg/phpdbg_prompt.c

```diff
@@ -107,23 +107,16 @@ static inline int phpdbg_call_register(phpdbg_param_t *stack) /* {{{ */
 		lc_name = zend_str_tolower_dup(name->str, name->len);
 
 		if (zend_hash_str_exists(&PHPDBG_G(registered), lc_name, name->len)) {
-			zval fretval;
-			zend_fcall_info fci;
-
-			memset(&fci, 0, sizeof(zend_fcall_info));
-
-			ZVAL_STRINGL(&fci.function_name, lc_name, name->len);
-			fci.size = sizeof(zend_fcall_info);
-			fci.object = NULL;
-			fci.retval = &fretval;
-			fci.param_count = 0;
-			fci.params = NULL;
-			fci.named_params = NULL;
+			HashTable *params_ht = NULL;
+			zend_function *user_fn = zend_hash_str_find_ptr(EG(function_table), lc_name, name->len);
+			if (user_fn == NULL) {
+				// TODO Debug for unknown function?
```

Comment:
Leak on lc_name

Reply:
Ah indeed, I don't really know how to write PHPDBG tests, but that would be a good one to cover

Reply:
See sapi/phpdbg/tests, it has a PHPDBG section. https://qa.php.net/phpt_details.php#phpdbg_section